src/login/templates/login.html

```diff
@@ -78,32 +76,46 @@ <h2 class="login-title">Telepy</h2>
   function get_jwt_using_google_credential(data) {
     const credential = data.credential;
     $.ajax({
-        method: "POST",
-        url: "/api/auth/google/token",
-        data: { credential: credential }
-    }).done(function (data) {
-        console.log(data);
-        const access_token = data.access;
-        const refresh_token = data.refresh_token;
-        localStorage.setItem('accessToken', access_token);
-        localStorage.setItem('refreshToken', refresh_token);
-        console.log('Google Login');
-        $.ajax({
-            type: "POST",
-            url: "/api/auth/token/verify",
-            data: {"token":access_token},
-            headers: {
-                "Authorization": "Bearer" + " " + access_token
-            },
-            success: function (data) {
-              console.log('Google Login Success');
-              //window.location.href = '/tunnels/index';
-            },
-            error: function (data) {
-                // Get `loginError` element and set the text to `login failed ${data}
-                $("#loginError").text(`Login failed ${data}`).css('color', 'red');
-            }
+      method: "POST",
+      url: "/api/auth/google/token",
+      data: { credential: credential },
+      success: function (data) {
+          console.log(data);
+          const access_token = data.access_token;
+          const refresh_token = data.refresh_token;
+          localStorage.setItem('accessToken', access_token);
+          localStorage.setItem('refreshToken', refresh_token);
+          console.log('Google Login');
+          $.ajax({
```

Comment:
It's Ajax. : (

Reply:
So pain 🥲

I've made a refactor of login function in this commit https://github.com/NatLee/telepy/pull/22/commits/ad2a422c816c22400c0a83f13ca3f23b9e9c2681